pkg/daemon/file_writers.go

```diff
@@ -227,6 +228,18 @@ func writeUnit(u ign3types.Unit, systemdRoot string, isCoreOSVariant bool) error
 				return err
 			}
 		}
+		// If the unit is currently enabled, disable it before overwriting since we might be
+		// changing its WantedBy= or RequiredBy= directive (see OCPBUGS-33694). Later code will
+		// re-enable the new unit as directed by the MachineConfig.
+		cmd := exec.Command("systemctl", "is-enabled", u.Name)
+		out, _ := cmd.CombinedOutput()
+		if cmd.ProcessState.ExitCode() == 0 && strings.TrimSpace(string(out)) == "enabled" {
+			klog.Infof("Disabling systemd unit %s before re-writing it", u.Name)
```

Comment:
[images][1]:

```
pkg/daemon/file_writers.go:237:4: undefined: klog
```

In 4.14, bcd40a10d21b1a47b302c9c9791bfbde03136d48 had moved everything from glog to klog, but back here in 4.13, we're still using glog.

[1]: https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_machine-config-operator/4522/pull-ci-openshift-machine-config-operator-release-4.13-images/1822034687002939392#1:build-log.txt%3A124

Reply:
Thanks!

Reply:
Addressed in f62bcee06b19; I'll mark this thread resolved.